static/css/page-signup.less

```diff
@@ -0,0 +1,156 @@
+/**
+ * Stylesheet for signup and (soon) login forms
+
+ * Used in templates:
+ * - openlibrary/templates/account/create.html
+ *
+ * Used on pages:
+ * - /account/create
+ */
+
+@import (less) "less/index.less";
+
+// Import all base styles
+@import (less) "base/index.less";
+
+// Import colors
+@import (less) "less/colors.less";
+
+// layout rules
+@import (less) "layout/index.less";
+@import (less) "page-user.less";
+
+// Components (modules) in order of appearance on page
+@import (less) "components/header.less";
+@import (less) "components/form.olform.less";
+@import (less) "components/footer.less";
+@import (less) "components/language.less";
+
+.ol-page-signup {
+  display: flex;
+  flex-direction: row;
+
+  .ol-signup-hero > *, .ol-signup-form > * {
+    width: 70%;
+  }
+
+  .ol-signup-hero {
```

Comment:
Same with this; this can be un-nested, and can collect everything that begins with `ol-signup-hero`